src/key-casing.ts

```diff
@@ -172,21 +282,47 @@ function deepDelimiterKeys<T, D extends string>(
     toDelimiterCase(str, delimiter),
   ) as never
 }
+/**
+ * A strongly typed function that shallowly the keys of an object to a custom delimiter case. The transformation is done both at runtime and type level.
```

Comment:
shallowly _ the keys

Reply:
fixed in [38c23df](https://github.com/gustavoguichard/string-ts/pull/16/commits/38c23dfc9096dcb745d8b0584f26d8e2e87a5fac)